main.cpp

```diff
@@ -52,85 +52,93 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
+//  Noun: Fork
+//  action 1: the fork holds food
+fork.holds();
+//  action 2: the fork pushes food forwards
+fork.pushes();
+//  action 3: the fork pulls food backwards
+fork.pulls();
 
-//  action 2:
-
-//  action 3:
- 
 //  2)
-//  Noun:
-//  action 1:
+//  Noun: Monkey
+//  action 1: the monkey climbs 
 
-//  action 2:
+//  action 2: the monkey sits
 
-//  action 3:
+//  action 3:the monkey falls asleep
  
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
+//  Noun: Cat 
+//  action 1: the cat meows
+cat.meow();
+//  action 2: the cat chases mice
+cat.chases();
+//  action 3: the cat annoys owner
+cat.annoys();
 
-//  action 3:
- 
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
+//  Noun: Bear
+//  action 1: the bear plays with sticks
+bear.playWithSticks();
+//  action 2: the bear destroys garbage cans
+bear.destroyGarbageCan();
+//  action 3: the bear looks for fish
+bear.looksForFish();
 
-//  action 3:
- 
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
+//  Noun: Stove
+//  action 1: the stove heats up
+stove.getHot();
+//  action 2: the stove lays still
+stove.layStill();
+//  action 3: the stove catches fire
+stove.catchFire();
 
-//  action 3:
- 
 //  6)
-//  Noun:
-//  action 1:
+//  Noun: Bicycle 
+//  action 1: the bicycle moves forward
+bicycle.moveForward();
+//  action 2: the bicycle moves backward
+bicycle.moveBackward();
+//  action 3: the bicycle stops moving
+bicycle.stopMoving();
 
-//  action 2:
-
-//  action 3:
- 
 //  7)
-//  Noun:
-//  action 1:
+//  Noun: SpeakerCone
+//  action 1: the speakercone reacts to current
+speakercone.reactsToCurrent();
+//  action 2: the speakercone lays still
+speakercone.layStill();
+//  action 3: the speakercone catches fire
+speakercone.catchFire();
```

Comment:
Speaker Cone is two words, so you should use `camelCaseWhenSpellingTheObjectName`
- [ ] revise